doc/diffs.md

```diff
@@ -468,3 +535,34 @@ diffing.  Note: The mscorlib.dll is removed, as this assembly should be updated
 the selected base runtime that is under test, for consistency. To add particular packages 
 to the set you diff, add their dependencies to the project.json in this project and 
 they will be pulled in and published in the standalone directory './fx'.
+
+## pmi
```

Comment:
I think PMI deserves its own file, since it doesn't need to be tied to assembly diffing.

Reply:
Moved it to a `Tools` page with the latest commit.